src/modules/InviteUser/components/invitationActions.vue

```diff
@@ -0,0 +1,153 @@
+<script>
+import { reInviteUser, revokeUser } from "../services";
+export default {
+    props: {
+        name: {
+            type: String,
+            required: true,
+        },
+        email: {
+            type: String,
+            required: true,
+        },
+        contact:{
+          type:String,
+          required:true
+        }
+        
+    },
+    methods: {
+      bodyData(){
+        return   {
+                  token:"1bxXVTgMmdgiClqXZ8Rdmg",
```

Comment:
Why are you using the static token, this should be extracted from the Backend API response.

Reply:
This static token is used for test api, it is to be replaced by response from backend after the backend server is hosted.